doc/docs/usage/tutorial-vehicle-signals.md

```diff
@@ -0,0 +1,324 @@
+# Tutorial: Sending and receiving vehicle signals
+
+## Introduction
+
+In this tutorial, we will show you how to use Ankaios to set up workloads that publish and subscribe to vehicle signals in accordance with the [Vehicle Signal Specification (VSS)](https://covesa.github.io/vehicle_signal_specification/).
+The central workload will be a databroker from the [Kuksa.val project](https://github.com/eclipse/kuksa.val). It will receive vehicle speed signals published from a speedometer workload. Finally a navigation workload will consume those speed limits.
+
+<figure markdown>
+  ![Overview of workloads](../assets/tutorial_vehicle_signals_overview.png)
+  <figcaption>Overview of workloads</figcaption>
+</figure>
+
+To run this tutorial you will need a Linux platform, which can be a RaspberryPi or a Linux PC or virtual machine.
+
+## Start the databroker
+
+If you have not yet installed Ankaios, please follow the instructions [here](https://eclipse-ankaios.github.io/ankaios/latest/usage/installation/).
+The following examples assume that the installation script has been used with the default options.
+
+Make sure that Ankaios server and agent are started:
+
+```shell
+sudo systemctl start ank-server
+sudo systemctl start ank-agent
+```
+
+Now we have Ankaios up and running with a server and an agent.
+To run the databroker we need to create an Ankaios manifest:
+
+```yaml title="databroker.yaml"
+workloads:
+  databroker:
+    runtime: podman
+    agent: agent_A
+    restart: true
+    updateStrategy: AT_MOST_ONCE
+    accessRights:
+      allow: []
+      deny: []
+    tags: []
+    runtimeConfig: |
+      image: ghcr.io/eclipse/kuksa.val/databroker:0.4.1
+      commandArgs: ["--insecure"]
+      commandOptions: ["--net=host"]
+```
+
+This defines a workload `databroker` to be scheduled on agent `agent_A` (default agent name when using standard installation procedure) using the runtime `podman`.
+See the [reference documentation](https://eclipse-ankaios.github.io/ankaios/latest/reference/startup-configuration/#configuration-structure) for the other attributes.
+
+Let's have a look at the `runtimeConfig` which in this case is specific for the `podman` runtime.
+
+* `image: ghcr.io/eclipse/kuksa.val/databroker:0.4.2` specifies the container image according to the OCI image format. Fortunately, the Kuksa.val project already provides an image for the databroker that we can use here.
```

Comment:
I think this is a typo here. In the runtimeConfig above you specify the tag `0.4.1` but here it is `0.4.2` which does not exist.

Reply:
I will change that to 0.4.1. I think the problem of pulling that image has been solved on you site for the time being.

Reply:
Yes, is resolved, it was an local issue.